topology/University of Campinas/CCJDR/feynman-PBS.yaml

```
@@ -0,0 +1,119 @@
+# Production is true if the resource is for production and not testing use
+Production: false
+
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Supported
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: This is a cluster of HPC used for the Dune
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: 1108
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  feynman:
```

```suggestion
  CCJDR-feynman:
```

Resource names need to be unique across the OSG so let's prefix it with your site name